src/hash-table.js

```diff
@@ -7,6 +7,25 @@ class HashTable {
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
+  insert(key, value) {
+    // Takes a key value pair and adds the value to the hash table
+    const index = getIndexBelowMax(key.toString(), this.limit);
+    this.storage[index] = value;
```

Comment:
You could consider storing both the key and the value for later resizing purposes.